platform/src/apis/Platform.Api.Establishment/Schools/SchoolsService.cs

```diff
@@ -26,7 +26,7 @@ public class SchoolsService(ISearchConnection<School> searchConnection, IDatabas
         using var conn = await dbFactory.GetConnection();
         var school = await conn.QueryFirstOrDefaultAsync<School>(schoolSql, new { URN = urn });
 
-        if (school != null && string.IsNullOrEmpty(school.FederationLeadURN))
+        if (school != null && !string.IsNullOrEmpty(school.FederationLeadURN))
         {
             school.Schools = await conn.QueryAsync<School>(childSchoolsSql, new { URN = urn });
```

Comment:
chore: To be explicit, please can this property be renamed FederatedSchools (or similar)

Reply:
It looks like this was added [here](https://github.com/DFE-Digital/education-benchmarking-and-insights/pull/1476/files#diff-3408b98052ad41104923ccc5753eb61bd369cb15278fea131979a105d9af43c5)

Reply:
Agreed 👍